New 1

jmoneybwa opened the pull request:
Fixes # @AndreMiras

XhmikosR committed:
Remove snyk.

XhmikosR committed:
Switch to `xo@0.24.0` for linting

Not using the latest version for Node.js 6.x support.

XhmikosR committed:
Update package.json

Remove the now unused pretest scripts

XhmikosR committed:
Fix Windows tests again.

XhmikosR committed:
CI: Add Windows testing and lint

XhmikosR committed:
Remove the now unused istanbul and mocha-lcov-reporter devDependencies

XhmikosR committed:
Revert #243.

cobertura-parse is archived and in bad shape. They even have mocha in dependencies.

XhmikosR committed:
CI: use `npm ci` on Node.js >=8.

XhmikosR committed:
Tweak README.md

* fix redirected links and use https when possible
* Markdown consistency changes

XhmikosR committed:
README.md: Add GitHub Actions CI info.

XhmikosR committed:
Fix logger regression.

nickmerwin committed:
fix equality operator in logger.js

nickmerwin committed:
Update logger.js

XhmikosR committed:
Revert "CI: use `npm ci` on Node.js >=8."

This reverts commit be79dab7f164da7762951fa5940dacdd3ec7ceff.

Too complex to make this work with full version numbers

nickmerwin committed:
bump version

XhmikosR committed:
package.json: reorder properties and add homepage

AndreMiras committed:
Only coverage report on test success, fixes #230

Addresses only the `jest` case, other cases can be handled via
subsequent pull requests.

suda committed:
Add Codefresh support

XhmikosR committed:
CI: switch to `actions/checkout@v2`

ashishkujoy committed:
278 Upgrades minimist dependency to 1.2.5

benpetty committed:
Set service_name and/or repo_token from .coveralls.yml regardless of if $COVERALLS_REPO_TOKEN is set (#272)

* using TRAVIS_COMMIT environment variable for git_commit

* Setting repo token and service name from .coveralls.yml regardless of if $COVERALLS_REPO_TOKEN

* only set options.repo_token from coveralls_yaml_conf if set in .coveralls.yml

* Update lib/getOptions.js

Co-Authored-By: Derek Herman <derek.herman@xwp.co>

nickmerwin committed:
bump version

midgleyc committed:
If COVERALLS_SERVICE_NUMBER is set, set service_number from it. (#208)

* If COVERALLS_SERVICE_NUMBER is set, set service_number from it.

Remove the duplicate service_pull_request set.

nickmerwin committed:
added COVERALLS_SERVICE_NUMBER to Readme; ver bump

nickmerwin committed:
Update getOptions.js

* Revert https://github.com/nickmerwin/node-coveralls/commit/3d83b4f29f2dbf6f4a642ba24976d4ecbd5d56b1#diff-55c2772ed44f17a0079fc5dabe37be63R29 to fix Travis fails due to missing git object

nickmerwin committed:
3.0.12

nickmerwin committed:
fix test

nickmerwin committed:
pass flag_name to API post JSON

nickmerwin committed:
3.0.13

lpinca committed:
remove redundant code (#283)

The check is already done above.

XhmikosR committed:
Minor lint tweaks (#285)

nickmerwin committed:
GH Action workflow update (#287)

* parallel
* flag-name

XhmikosR committed:
Move fixtures to test/fixtures. (#286)

XhmikosR committed:
Update safe dependencies and fix npm vulnerabilities (#284)

nickmerwin committed:
3.0.14

XhmikosR committed:
Update sinon to v7.5.0. (#288)

Co-authored-by: Xavier <yadd@debian.org>

nickmerwin committed:
Handle service_job_number for parallelism in Travis and CircleCI (#290)

* support `service_job_number` for CircleCI/Travis
* Travis use TRAVIS_JOB_ID

nickmerwin committed:
3.1.0

nickmerwin committed:
readme re: flag names, CircleCI Orb

nickmerwin committed:
update README with links to demo repo

nickmerwin committed:
readme typo

XhmikosR committed:
Minor CI tweaks

* add Node.js 14
* rename job to test
* uppercase `GITHUB_TOKEN` for consistency
* remove no longer needed steps; `actions-setup-node` prints this by default now

XhmikosR committed:
tests: fix deprecation warning coming from `sinon.sandbox.create()`

Switch to the default sandbox.

XhmikosR committed:
Update sinon to v.8.x.

The latest sinon version (9.x) drops support for older Nojde.js versions.

XhmikosR committed:
Regenerate package-lock.json

XhmikosR committed:
xo: make `handle-callback-err` an error.

This will prevent tests not checking for errors.

XhmikosR committed:
Update README.md

* use an unordered list for the supported CI services and make it a heading
* fix tap repo URL
* add a few trailing slahes in URLs
* remove a duplicate trailing slash
* minor grammar tweaks

jmoneybwa commented:
Erica

dependabot[bot] committed:
Bump ini from 1.3.5 to 1.3.7

Bumps [ini](https://github.com/isaacs/ini) from 1.3.5 to 1.3.7.
- [Release notes](https://github.com/isaacs/ini/releases)
- [Commits](https://github.com/isaacs/ini/compare/v1.3.5...v1.3.7)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump y18n from 4.0.0 to 4.0.1

Bumps [y18n](https://github.com/yargs/y18n) from 4.0.0 to 4.0.1.
- [Release notes](https://github.com/yargs/y18n/releases)
- [Changelog](https://github.com/yargs/y18n/blob/master/CHANGELOG.md)
- [Commits](https://github.com/yargs/y18n/commits)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump lodash from 4.17.15 to 4.17.21

Bumps [lodash](https://github.com/lodash/lodash) from 4.17.15 to 4.17.21.
- [Release notes](https://github.com/lodash/lodash/releases)
- [Commits](https://github.com/lodash/lodash/compare/4.17.15...4.17.21)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump hosted-git-info from 2.8.8 to 2.8.9

Bumps [hosted-git-info](https://github.com/npm/hosted-git-info) from 2.8.8 to 2.8.9.
- [Release notes](https://github.com/npm/hosted-git-info/releases)
- [Changelog](https://github.com/npm/hosted-git-info/blob/v2.8.9/CHANGELOG.md)
- [Commits](https://github.com/npm/hosted-git-info/compare/v2.8.8...v2.8.9)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump dot-prop from 4.2.0 to 4.2.1

Bumps [dot-prop](https://github.com/sindresorhus/dot-prop) from 4.2.0 to 4.2.1.
- [Release notes](https://github.com/sindresorhus/dot-prop/releases)
- [Commits](https://github.com/sindresorhus/dot-prop/compare/v4.2.0...v4.2.1)

---
updated-dependencies:
- dependency-name: dot-prop
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

nickmerwin committed:
swap exec for execFile in fetchGitData

Credit: Adar Zandberg from the CxSCA AppSec team at Checkmarx.

* devDependency updates from Dependabot alerts
* bump version

jtwebman commented:
Since this library doesn't seem to be supported anymore I fix a bunch of things on a fork if you want to check it out and are still pulling the library into your packages: https://github.com/jtwebman/coveralls-next

afinetooth committed:
Add maintenance notice and recommended alternatives

Added important notice about repository maintenance status and recommended alternatives.

nickmerwin committed:
Merge pull request #339 from afinetooth/deprecation-notice

Add maintenance notice and recommended alternatives
